db/migration/V017__add_categories_and_competencies_data.sql

```diff
@@ -0,0 +1,72 @@
+    ('Operational', 'Competencies used for daily management of tasks and relationships; the application of knowledge, skills, experience, and behaviours to make competencies operational in work situations', 'https://opentree-education.imgix.net/competency/CompetencyGraphic_Operational.png'),
+    ('Behavioural', 'Competencies to achieve results through self-awareness and, accordingly, by working effectively with others in various circumstances; the ability to develop oneself, initiate action, self-discipline, self-esteem, individuality, and self-determination', 'https://opentree-education.imgix.net/competency/CompetencyGraphic_Behavioural.png'),
+    ('Organizational', 'Competencies to communicate by various means within different organizational settings; applied knowledge of language and its use in an effective way', 'https://opentree-education.imgix.net/competency/CompetencyGraphic_Organizational.png');
+
+
+INSERT INTO competencies (label, description, principal_id, category_id) VALUES
+    ('Data Structures', 'Demonstrates knowledge of data structures, or formats for organizing, processing, retrieving and storing data; can effectively use and explain arrays, linkedlists, dictionaries, etc., in practical programming tasks.', @principalId, 1),
+    ('System Design', 'Demonstrates understanding of the process of designing the elements of a system, including architecture, modules and components, interfaces and data.', @principalId, 1),
+    ('Source Code and Version Control', 'Uses, and leverages, version control systems and their features, in addition to branching and merging.', @principalId, 1),
+    ('Testing', 'Skillfully uses the appropriate tests as first line of defense for catching bugs and tracing requirements; includes unit and/or integration tests.', @principalId, 1),
+    ('Problem and Systems Decomposition', 'Breaks down problems and/or systems so that significant numbers of subproblems are defined at smaller, more manageable levels.', @principalId, 1),
+    ('Code Quality and Readability', 'Writes good quality code that is designed to be maintained.', @principalId, 1),
+    ('Error Handling', 'Anticipates, detects, and resolves programming, application, and communication errors effectively and efficiently.', @principalId, 1),
+    ('Requirements', 'Documents and describes that specific software will do and how it is expected to perform; includes defining a product purpose, describing the build, detailing the requirements, and delivering for approval.', @principalId, 1),
+    ('Debugging', "Addresses and removes errors in a program's coding language; uses debugging software and exercises fundamental strategies for removing errors and developing an effective debugging process.", @principalId, 1),
+    ('Secure Programming Mindset', 'Executes secure coding practices with attention to confidentiality, integrity, and availability (CIA triangle); optimizes security.', @principalId, 1),
+
```

Comment:
Let's remove these extra whitespace lines. If you prefer some organizational separation of the categories then 1 line is sufficient.